pkg/apis/crd/v1beta1/types.go

```diff
@@ -381,6 +381,11 @@ type GroupSpec struct {
 	// Cannot be set with any other selector except PodSelector.
 	// +optional
 	NamespaceSelector *metav1.LabelSelector `json:"namespaceSelector,omitempty"`
+	// Select all Pods from Nodes matched by this selector, as
+	// workloads in AppliedTo fields.
+	// TODO: put in limitations on the usage NodeSelector? Should we support combining with ns/pod selector? if not, add guardrails/feedback?
```

Comment:
Yes on the TODO. Refer to https://github.com/antrea-io/antrea/blob/main/pkg/controller/networkpolicy/validate.go#L1049